src/pages/home-page.html

```diff
@@ -18,10 +22,136 @@
 
       :host {
         display: block;
+        height: 100%;
+      }
+
+      hero-block {
+        height: calc(100% + 56px);
+        font-size: 24px;
+        text-align: center;
+      }
+
+      .home-content {
+        margin-top: -48px;
+      }
+
+      .hero-logo {
+        --iron-image-height: 72px;
+      }
+
+      .info-items {
+        margin-top: 32px;
+        line-height: 1.1;
+      }
+
+      .hero-description {
+        margin-top: 4px;
+      }
+
+      .action-buttons {
+        margin-top: 36px;
+        font-size: 14px;
+      }
+
+      .action-buttons iron-icon {
+        --iron-icon-fill-color: currentColor;
+        margin-right: 8px;
+      }
+
+      .scroll-down {
+        position: absolute;
+        bottom: 24px;
+        left: 50%;
+        transform: translateX(-50%);
+        z-index: 2;
+        color: currentColor;
+        user-select: none;
       }
 
+      .scroll-down svg {
+        width: 24px;
+        opacity: 0.6;
+      }
+
+      .scroll-down .stroke {
+        stroke: currentColor;
+      }
+
+      .scroll-down .scroller {
+        fill: currentColor;
+        animation: updown 2s infinite;
+      }
+
+      @keyframes updown {
+        0% {
+          transform: translate(0, 0)
+        }
+        50% {
+          transform: translate(0, 5px)
+        }
+        100% {
+          transform: translate(0, 0)
+        }
+      }
+
+      @media (min-width: 640px) {
+        hero-block {
+          height: calc(100% + 64px);
```

Comment:
same `100vh;`

Reply:
I wish it works, but mobile Safari + now mobile Chrome doesn't handle it correctly, so I had to add a bunch of `height: 100%` through the app.

Reply:
strange 😢, according to https://caniuse.com/#search=vh , it should work

Reply:
It is supported but in their way :)
More info here: https://stackoverflow.com/a/37113430